pom.xml

```diff
@@ -29,8 +29,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <!-- DependencyGraph is final in Jenkins < 1.425 -->
-        <version>1.425</version>
+        <version>1.532.3</version>

```

Comment:
Though I know it's not the important point we discuss, I'd like to let you know you can use java 6 even with 1.425 at compilation time.
In fact, I use annotations (which is introduced in Java 6) in flexible-publish.
Older Jenkins is compiled with Java 6, and can run on Java 5.